cmd/internal/cli/singularity.go

```diff
@@ -133,7 +133,7 @@ func getDefaultTokenFile() string {
 // have been properly loaded and initialized
 func initializePlugins() {
 	if err := plugin.InitializeAll(buildcfg.LIBEXECDIR); err != nil {
-		sylog.Fatalf("Unable to initialize plugins: %s\n", err)
+		sylog.Warningf("Unable to initialize plugins: %s", err)
```

Comment:
Why is this a Warning now? What if a user doesn't see this warning and is confused why their plugin is not working, should this be `Fatalf`?

Reply:
We can't use `Fatal` b/c if there is a broken plugin installed `singularity` won't be able to run and e.g. uninstall it. 
This is a very basic fix for that, we will be working on plugin recompilation.

And yes, it is still WIP. Need everyone's opinion.

Reply:
Makes sense, but then how about using `sylog.Errorf`? This seems like its more of an error, then a warning.